apps/antalmanac/src/components/Calendar/CalendarRoot.tsx

```diff
@@ -163,9 +163,13 @@ export default function ScheduleCalendar(props: ScheduleCalendarProps) {
     const calendarTimeFormat = isMilitaryTime ? 'HH:mm' : 'h:mm A';
     const calendarGutterTimeFormat = isMilitaryTime ? 'HH:mm' : 'h A';
 
-    const defaultFinals = getDefaultFinalsStartDate();
-    const finalsDateFormat = defaultFinals ? 'ddd MM/DD' : 'ddd';
-    const date = showFinalsSchedule && defaultFinals ? defaultFinals : new Date(2018, 0, 1);
+    const finalsDate =
+        eventsInCalendar.length > 0 ? getFinalsStartDateForTerm(eventsInCalendar[0].term) : getDefaultFinalsStartDate();
```

Comment:
If the first event or only event in the calendar is a custom event, this would not work since custom events don't have a term?

If the first thing I add to a schedule is a custom event, I get this, probably because `getFinalsStartForTerm` has an undefined term and tried to call `finalsStartDate` on it:
![image](https://github.com/icssc/AntAlmanac/assets/78244965/fa0390d1-581e-4fe8-9a17-273e5653a789)